manifests/bucketeer/values.dev.yaml

```
@@ -73,6 +73,12 @@ backend:
       clientSecret:
       redirectUrls:
         - https://localhost:9000/auth/callback
+    demoSignInConfig:
+      email: demo@bucketeer.io
+      password: demo
+      organization: demo
+      project: default
+      environment: default
```

Please update the other places accordingly.

```suggestion
      organizationId: demo
      projectId: demo
      environmentId: demo
```